src/core/status-line/ansi.ts

```
@@ -0,0 +1,33 @@
+export function color(text: string, colorCode: string): string {
+  if (!colorCode) return text;
+  return `\x1b[${colorCode}m${text}\x1b[0m`;
+}
+
+/**
+ * Strip all ANSI escape codes from a string.
+ * Useful for testing and calculating visible string length.
+ */
+export function strip(text: string): string {
+  // eslint-disable-next-line no-control-regex
+  return text.replace(/\x1b\[\d+m/g, "");
```

Biome (2.4.4)

[error] Unexpected control character in a regular expression.

(lint/suspicious/noControlCharactersInRegex)